apps/odyssey-stats/src/lib/selectors/get-site-stats-base-url.ts

```diff
@@ -0,0 +1,10 @@
+import isSiteAutomatedTransfer from 'calypso/state/selectors/is-site-automated-transfer';
```

Comment:
Should we be using this or `client/state/selectors/is-site-wpcom-atomic.js`? What's the difference? I noticed we added both to the corresponding Jetpack PR; do we need both?

Reply:
@jsnmoon Good question!

I think these two are the same thing, but both are widely used in Calypso. I added two options just to make sure all the components in Calypso would work as expected when we use them.

The reason I chose `is-site-automated-transfer` is because it's more frequently used in Calypso than `is-site-wpcom-atomic`.

`is-site-automated-transfer` is determined by

https://github.com/Automattic/jetpack/blob/c76f8f4572d1c59b1c6c3d7fdb76347288f503af/projects/packages/status/src/class-host.php#L40

while `is-site-wpcom-atomic` is filtered and forced to true on Atomic sites:

```
wp-content/mu-plugins/wpcomsh/wpcomsh.php:252:add_filter( 'jetpack_site_automated_transfer', '__return_true' );
```

However, they could have very slight differences and we don't want that to affect any of our functionalities, so I'll open another Jetpack PR to align that and use `is-site-wpcom-atomic` which would seem more explicit.

Reply:
Changed in b043399072

Reply:
Thanks for the clarification! 👍